src/pages/EditMonitor.vue

```diff
@@ -1120,6 +1134,7 @@ message HealthCheckResponse {
                     url: "https://",
                     method: "GET",
                     interval: 60,
+                    timeout: this.interval * 0.8, // previous default value
```

Comment:
Previous compared to what?
I think this comment will not age well ^^

```suggestion
                    timeout: this.interval * 0.8,
```